todo_app/data/Trello.py

```diff
@@ -79,8 +85,8 @@ def get_list_items():
         items = []
         for card in cards:
             status = search_list(lists, 'id', card['idList'])['name']
+            items.append(TrelloItem(card['id'], card['name'], status, card['dateLastActivity']))
```

Comment:
Your code assumes that the `date_last_activity` field is a datetime, but here you just pass through the string; that means that your app fails to display Done cards if there are more than 5, as the `recent_done_items` property errors. I think the only change required is to use your function on the string to conver that, e.g.:
```python
items.append(TrelloItem(card['id'], card['name'], status, to_utc_datetime(card['dateLastActivity'])))
```